templates/download/iot/intel-iotg.html

```diff
@@ -0,0 +1,71 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Download Ubuntu for Intel IoT platforms | Download{% endblock %}
```

Comment:
Could you create a social banner for this page using https://tools.demos.haus/. You can see what this page looks like on social media here: https://www.opengraph.xyz/url/https%3A%2F%2Fubuntu-com-10332.demos.haus%2Fdownload%2Fiot%2Fintel-iotg/

Reply:
Nice :+1: